navit/CMakeLists.txt

```diff
@@ -124,48 +124,93 @@ add_custom_command(
 	)
 
 # additional parameter are passed to the stylesheet processor as parameter
-macro(process_xslt SRC_XML DEST_XML)
-	set(XSLT_COMMANDS COMMAND ${CMAKE_COMMAND} -E copy "${PROJECT_SOURCE_DIR}/navit/navit_shipped.xml" "${CMAKE_CURRENT_BINARY_DIR}/${DEST_XML}" )
+macro(process_xslt SRC_DIR SRC_XML DEST_DIR DEST_XML)
+	if(WIN32)
+		set(TMPDIR "$ENV{TEMP}")
+	else()
+		set(TMPDIR "/tmp")
```

Comment:
Suggest changing this to
`set(TMPDIR "${CMAKE_CURRENT_BINARY_DIR}")`
to put intermediate files into CMAKEs build directory. Putting it hard into /tmp is preventing parallel build runs, and somehow uncommon. It should all be placed in CMAKEs build dir shouldn't it?
And this prevents the rename from failing if /tmp is on a different filesystem than the build directory.

Reply:
Yes, sure, let me try that.
I am also considering changing the gradle build scripts to use the cmake target for xslt processing (rather that duplicating the code)

Reply:
Build for android now uses cmake for xslt processing (was implemented https://github.com/navit-gps/navit/pull/559/commits/4e1a6f2314380c0fc5c7d52634de69bead6ee490)